src/queryResult/formatter.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import * as prettier from "prettier";
+
+export class JsonFormattingEditProvider implements vscode.DocumentFormattingEditProvider {
```

Comment:
For better code organization, this probably go in the `src/utils` folder, and either be called "jsonFormatter.ts" or (if we also add an xml formatter here) formatter_s_.ts.